src/data/markdown/docs/05 Examples/01 Examples/09 data-uploads.md

```diff
@@ -86,3 +86,52 @@ and sent.
 
 - [open(filePath, [mode])](/javascript-api/init-context/open-filepath-mode)
 - [http.file(data, [filename], [contentType])](/javascript-api/k6-http/file-data-filename-contenttype)
+
+
+## Advanced multipart request
+
+The previous multipart request example has some limitations:
+
+- It's not possible to assemble the parts in a specific order, because of the
+  unordered nature of JS objects when they're converted to Golang maps, which k6 uses internally.
+  Uploading files in a specific order is a requirement for some APIs (e.g. AWS S3).
+- It's not possible to upload multiple files as part of the same form field, because
+  JS object keys must be unique.
+
+To address this we suggest using the [`FormData` polyfill for k6](https://jslib.k6.io/formdata/0.0.1/index.js).
+
+Here's an example of uploading several binary files and a text file using the polyfill:
+
+<div class="code-group" data-props='{ "labels": ["multipart-upload-formdata.js"], "lineNumbers": [true] }'>
+
+```javascript
+import http from 'k6/http';
+import { check } from 'k6';
+import { FormData } from 'https://jslib.k6.io/formdata/0.0.1/index.js';
+
+const img1 = open('/path/to/image1.png', 'b');
+const img2 = open('/path/to/image2.jpg', 'b');
+const txt = open('/path/to/text.txt');
+
+export default function() {
+  let fd = new FormData();
```

Comment:
nit: this can also be `const` as `const` is only about the identifier - it prevents the identifier to point/identify another object, after initialization.

Reply:
Thanks, I was aware of that, but wasn't thorough enough with the usage. Will change it.

Reply:
Resolved in 397babd.